app/mainwindow2.cpp

```diff
@@ -380,6 +381,13 @@ void MainWindow2::updateSaveState()
     setWindowModified( mEditor->currentBackup() != mBackupAtSave );
 }
 
+void MainWindow2::clearRecentFilesList()
+{
+    mRecentFileMenu->clear();
+    InformativeDialog *informDialog = new InformativeDialog("Notice!", "You have sucessfully cleared the list");
+    informDialog->exec();
```

Comment:
Personally, I’d find it more intuitive to simply enable/disable the button depending on whether the list is empty (already clear) or not, though a dialog is certainly more explicit.